viewmodel-editor/viewmodel-editor.js

```diff
@@ -213,6 +216,11 @@ export default class ViewmodelEditor extends StacheElement {
 		this.dispatch("reset-json-patches");
 	}
 
+	reset() {
+		this.dispatch("reset-json-patches");
+		this.dispatch("serializedViewModelData", [this.viewModelData]);
```

Comment:
This doesn't seem like the right way to update `json`; since `serializedViewModel` is a getter it seems odd to dispatch this event in order to trigger the listener in the `json` resolver.

I think it would make more sense to have this code: https://github.com/canjs/can-devtools-components/pull/100/files#diff-c0544349455fdac8c5e130d97a1f7ac4R130-R138 called inside of https://github.com/canjs/can-devtools-components/pull/100/files#diff-c0544349455fdac8c5e130d97a1f7ac4R123-R125

Reply:
I've updated it so `reset` is now just `this.jsonEditorPatches = [];`.